src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +132,17 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sort all persons in list in alphabetical order, case insensitive.
+     */
+    public void sortAlphabet() {
+        Collections.sort(internalList, (firstPerson, secondPerson) -> {
```

Comment:
You can also consider implementing a separate class that implements the Comparator interface. By doing this you would be implementing abstraction on the side as well